apps/explorer/.eslintrc.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
     plugins: ['header'],
     extends: ['react-app', 'prettier'],
     rules: {
-        'react/jsx-no-bind': ['error'],
```

Comment:
I removed this because I don't think this rule is actually useful, generally creating functions during render and passing them as props is fine, short of passing to memo'd components, which is a rare case.